Board Meeting Minutes/CCC Board Minutes 2020-01-16.md

```diff
@@ -0,0 +1,67 @@
+**Governing Board Conference Call -- 7:00am PST**
+
+**16 January 2020**
+
+1.  **Call to Order / Roll Call**
+
+    a.  **Premier Members of the CCC**
+
+        i.  Philippe Robin (Arm)\*
+
+        ii. Sergio Leunissen (Oracle)\*
+
+        iii. Mike Bursell (Red Hat)\*
+
+        iv. Stephen Walli (Microsoft)\* Chair
+
+        v.  Ron Perez (Intel)*
+
+        vi. Dave Thaler (Microsoft) (TAC Chair)
```

Comment:
The 02-13 and 03-12 minutes do not have an * after my name, we should be consistent.
Per the bylaws as they stand, the TAC chair is a voting member and should have an *.
However, by TAC vote 10-31-2019 the TAC chair shall abstain from all votes unless/until the bylaws are amended to make the TAC chair be a non-voting member.  So until such amendment is approved, we need a * and a note on every vote that the TAC chair abstains.
```suggestion
        vi. Dave Thaler (Microsoft)* (TAC Chair)
```